contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2750,16 +2750,21 @@ rewriteBatchLevelStatement(
 	else if (ctx->create_or_alter_procedure())
 	{
 		auto cctx = ctx->create_or_alter_procedure();
+		size_t num_commas_in_procedure_param = cctx->COMMA().size();
```

Comment:
Let's move this into the below `if (cctx->WITH())` because it is only needed when `WITH` exists.

Reply:
Agree with you, will fix in next revision.